docs/source/jupyter/sdk.ipynb

```diff
@@ -67,12 +69,11 @@
     }
    ],
    "source": [
-    "from typing import List, Dict\n",
-    "from marketplace.app import MarketPlaceApp\n",
-    "\n",
-    "mp = MarketPlaceApp(client_id=\"<registered application's client_id>\",\n",
-    "                    access_token=\"<your access token>\")\n",
-    "\n",
+    "from marketplace.app import get_app\r\n",
+    "\r\n",
+    "mp = get_app(app_id=\"<registered application's client_id>\",\r\n",
+    "                    access_token=\"<your access token>\")\r\n",
```

Comment:
So, that would no longer be the recommended approach, right? I would suggest that we mention that we assume that the access token is stored in an environment variable (`$MP_ACCESS_TOKEN`), and then show how to create a client in case that is not the case.